addons/website_slides/controllers/main.py

```diff
@@ -137,6 +137,7 @@ def _get_slide_quiz_partner_info(self, slide, quiz_done=False):
 
     def _get_slide_quiz_data(self, slide):
         is_designer = request.env.user.has_group('website.group_website_designer')
+        slides_resources = slide.slide_resource_ids if slide.channel_id.is_member else []
```

Comment:
Or `can_publish`, which grants access to superuser and course responsible, iirc.

Reply:
You're right. The info is fetched again after submission.